src/prisma/cli/_node.py

```diff
@@ -0,0 +1,320 @@
+from __future__ import annotations
+
+import os
+import sys
+import shutil
+import logging
+import subprocess
+from pathlib import Path
+from abc import ABC, abstractmethod
+from typing import TYPE_CHECKING, IO, Union, Any, Mapping, cast
+from typing_extensions import Literal
+
+from pydantic.typing import get_args
+
+from .. import config
+from .._proxy import LazyProxy
+from ..binaries import platform
+from ..errors import PrismaError
+
+if TYPE_CHECKING:
+    # TODO
+    import nodejs  # type: ignore
```

Comment:
I have a WIP to add typings to this project for us: https://github.com/jacobdr/nodejs-pypi/pull/4 but haven't heard from the maintainer yet

Reply:
Thanks! That PR should only effect internal typing right?

The package was previously missing a `py.typed` file which is the original reason for these `# type: ignore`s. I [created a PR](https://github.com/samwillis/nodejs-pypi/pull/8) to add it which was merged and released so these may be unnecessary now.

Reply:
Did we turn on the mypy setting to turn unused ignores into errors? That would answer that question. And with re: to external types, yeah its only a partial solution so far, but eventually I'll add external types to that project

Reply:
Yeah the unused ignores setting should be turned on, I think the issue may be that we installing an outdated version